scripts/config_template.yaml

```diff
@@ -21,3 +21,10 @@ timeframe:
   start: 2024-02-01
   end: 2024-04-30
   interval: 7d
+
+
+# The options to return projection or evaluation metrics, can be "projection" or "evaluation"
+option: projection
```

Comment:
I think you want both? Like, I'm never going to want *just* scores, I'm always going to want to see scores *and* some diagnostic plots

Also, don't you need the projections, to do the scores? So one of these options is "do the projections, get the scores, and throw the projections away"

I think this is where you'd be better off having a workflow that makes the projections, and then builds off that to do the evaluation. You can code up that workflow in a Makefile

Reply:
I agree with @swo. Instead of having options at all, I recommend just one workflow that creates projections and gives the desired scores. 

Specifically, your `interval_number` in main.py line 69 could be set by the user in the config, rather than taken from `config["timeframe"]["interval"]`. This would allow the user to customize the forecasts independently of the time intervals in the data. That is, data points are reported every week, but your forecast dates could be, e.g., monthly with forecasts starting on Feb 1, Mar 1, Apr 1, etc. If the user doesn't enter a value for this in the config, the default behavior could be to just generate one forecast and evaluation, without varying the forecast date at all.

I like having one script, `main.py`, that runs the entire pipeline every time, because this helps me check that all parts of the pipeline cooperate with each other. But ultimately the Makefile will be better because it won't repeat stuff that's already been done and is up-to-date. Perhaps we should work together on such a Makefile in a separate PR.

Reply:
I will leave this in a future PR.

Reply:
Will leave this to the next PR.

Reply:
Punting to #89